### PR TITLE
[CERTUTIL] Add skeleton application with support for -hashfile

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

**Comment by HBelusca** [comment]
Out of curiosity, are there programs that use this certutil?

**Comment by learn-more** [comment]
> Out of curiosity, are there programs that use this certutil?

I wanted to check a file hash in ros, and it was implementing this or sha1sum.
Considering windows ships this, and not sha1sum I decided on this.

File: base/applications/certutil/certutil.c [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            DWORD cbRead;
+
+            while ((bSuccess = ReadFile(hFile, Buffer, sizeof(Buffer), &cbRead, NULL)))
+            {
+                if (cbRead == 0)
+                    break;
+
+                if (!CryptHashData(hHash, Buffer, cbRead, 0))
+                {
+                    bSuccess = FALSE;
```
Comment (ThFabba) [inline base/applications/certutil/certutil.c:45]: `break` in this case, too?

File: base/applications/certutil/certutil.c [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            {
+                if (!_wcsicmp(argv[n+1], L"-?"))
+                {
+                    print_usage();
+                    return EXIT_SUCCESS;
+                }
+                else
+                {
+                    BOOL bSuccess = hash_file(argv[n+1]);
+                    ConPrintf(StdOut, L"CertUtil: -hashfile command %s\n", bSuccess ? L"completed successfully" : L"FAILED");
```
Comment (ThFabba) [inline base/applications/certutil/certutil.c:131]: You're already printing the failure message in all failure cases, so you'll get double
```
>base\applications\certutil\certutil.exe -hashfile C:\bootnx
CertUtil: -hashfile command failed: 2
CertUtil: -hashfile command FAILED
```

File: base/applications/certutil/certutil.c [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                    return bSuccess ? EXIT_SUCCESS : EXIT_FAILURE;
+                }
+            }
+            else
+            {
+                ConPuts(StdOut, L"Expected no more than 1 args for -hashfile\n");
```
Comment (ThFabba) [inline base/applications/certutil/certutil.c:137]: This also prints for when no arguments are given to `-hashfile`, in which case the message is a little confusing.
```
>base\applications\certutil\certutil.exe -hashfile
Expected no more than 1 args for -hashfile
```

File: base/applications/certutil/certutil.c (hunk 2 above)
Comment (learn-more) [reply base/applications/certutil/certutil.c:131]: yes, following windows output (more or less)

File: base/applications/certutil/certutil.c (hunk 3 above)
Comment (learn-more) [reply base/applications/certutil/certutil.c:137]:
```
C:\Users\Mark.DEV2>certutil -hashfile
Expected at least 1 args, received 0
CertUtil: Missing argument
...
```

**Pull request merged**